Sources/ClientRuntime/Identity/IdentityAPI/Identity.swift

```diff
@@ -11,3 +11,8 @@ import Foundation
 public protocol Identity {
     var expiration: Date? { get }
 }
+
+// Enum of identity types supported by SDK
+public enum IdentityType: CaseIterable {
```

Comment:
If this code will have Identity, IdentityType, IdentityT we need a lot more comments so it is clear to the reader what each is for without having to read through the code

Reply:
This change is related to auth schemes, removed from this PR